src/App.js

```diff
@@ -1,5 +1,44 @@
+import { createContext, useState, useEffect } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { collection, getDocs } from "firebase/firestore";
+
+import { db } from "./firebase-config";
+import Navbar from "./components/Navbar";
+import Login from "./components/Login";
+import Users from "./components/Users";
+import Expense from "./components/Expense";
+import CreateExpense from "./components/Expense/CreateExpense";
+
+export const userContext = createContext();
+
 const App = () => {
-  return <div>App</div>;
+  const [users, setUsers] = useState([]);
+  const usersCollectionRef = collection(db, "users");
+
+  useEffect(() => {
+    const getUserData = async () => {
+      const data = await getDocs(usersCollectionRef);
+      setUsers(data.docs.map((doc) => ({ ...doc.data(), id: doc.id })));
+      console.log(users);
```

Comment:
Are we okay to have the console in our project?

Reply:
We can remove this console. :+1: